### ISSUE TITLE
Mismatched gem dependencies on tty-screen

### ISSUE THREAD

**Issue by floatplane** [peter-murach/tty-progressbar#10]
Hi! Thanks for providing this gem. I wanted to let you know that c3e2221d causes problems when installing the tty gem, producing output like the following:

```
ERROR:  While executing gem ... (Gem::DependencyError)
    Unable to resolve dependencies: tty requires tty-screen (~> 0.1.0); tty-progressbar requires tty-screen (~> 0.2.0); tty-table requires tty-screen (~> 0.1)
```

Is it possible to get them all in sync? I can also resolve this by updating my gem file to require a previous version of tty, but I wasn't sure if you'd seen this error.

**Comment by peter-murach** [comment]
Hi Brian! I will resolve this issue by bringing all dependencies in sync and release new tty version - hold tight.

**Comment by peter-murach** [comment]
Sorry forgotten to let you know about the release. I've released `tty` version `0.2.0` that addresses this issue. Give it a try and let me know if it works for you.
